client/keys.go

```diff
@@ -104,6 +104,9 @@ type KeysAPI interface {
 	// Get retrieves a set of Nodes from etcd
 	Get(ctx context.Context, key string, opts *GetOptions) (*Response, error)
 
+	// GetVersion retrieves the current etcdserver and etcdcluster version
+	GetVersion(ctx context.Context) ([]byte, error)
```

Comment:
getversion should not be inside keys API. keys API is for key operations like put, get, delete.